docs/beamrunner/beamrunner-6-issues.md

```diff
@@ -19,6 +19,11 @@ Learn about limitations and known issues for IBM® Streams Runner for Apache Bea
 - Experimental Splittable ParDos are not supported.
 
 ## Known issues
+### Issue
+Submission-time parameters specified while manually submitting SAB file (created via `BUNDLE` context) are not used during application runtime. 

```

Comment:
Change to:
Submission-time parameters that are specified when you manually submit an SAB file (created by using the `BUNDLE` context) are not used during the application runtime.